consensus/src/epoch_manager.rs

```diff
@@ -680,16 +701,19 @@ impl EpochManager {
 
         self.epoch_state = Some(epoch_state.clone());
 
-        let initial_data = self
-            .storage
-            .start()
-            .expect_recovery_data("consensusdb is not consistent with aptosdb");
-        self.start_round_manager(
-            initial_data,
-            epoch_state,
-            onchain_config.unwrap_or_default(),
-        )
-        .await;
+        match self.storage.start() {
+            LivenessStorageData::FullRecoveryData(initial_data) => {
+                self.start_round_manager(
+                    initial_data,
+                    epoch_state,
+                    onchain_config.unwrap_or_default(),
+                )
+                .await
+            }
+            LivenessStorageData::PartialRecoveryData(ledger_data) => {
+                self.start_recovery_manager(ledger_data, epoch_state).await
```

Comment:
It would be helpful to add a log message here with something like - validator is starting up in recovery mode and will restart once recovery finishes.

Reply:
there's a log message when start the recovery manager